travis-test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -e
+
+dotnet test -c Debug -f netcoreapp1.0 MoreLinq.Test
+RID=`dotnet --info | grep RID | sed -E 's/ *RID: *(.+)/\1/'`
```

Comment:
…point being, it seems to work for non-Windows platforms if you rely on the RID reported by `dotnet --info` but it could be that it might have to be eventually picked up dynamically by just scanning the directory below `net451`. Meanwhile, I'm hoping this is just a kink due to the tooling being preview.